### ISSUE TITLE
Links in the Admin Panel and Recent Activity page don't take me to the adventure

### ISSUE THREAD

**Issue by MattColville** [AdventureLookup/AdventureLookup#136]
I can look at the list of Change Requests, but I can't actually get to the adventure from there. There's a link for the adventure, but when I click on it I get an uneditable list of fields and data which...not sure what I'm supposed to do with that. :D

Better if the link in the Change Request listing took me to the adventure page so I could edit it and resolve the request.

Ditto the list in the Recent Activity. There are links, but they don't take me to the adventure so I can't look at the adventure that was just edited.

**Comment by MattColville** [comment]
Also, I can't click on a user and see their submissions or changes. This is critical to making new curators, I need to see what stuff (if anything) they've done.

Really I might only need to see adventures they've added and change requests they've submitted?

**Comment by harkenn** [comment]
Just got around to looking at this. Looks like I'll need to write a custom action since Easy Admin can't handle this very well on its own.

**Comment by MattColville** [comment]
I think we need a better curator page. It's sort of a pain in the ass right now.

Curators need to be able to easily see a list of adventures that need work. This will enable folks who own the adventure to recognize it and say "Oh, I have this! I can help!"

As it stands you can sort by adventure, but it shows you adventures with resolved change requests. So The Lost City shows up, even though it has no pending change requests. Not sure why this should be.

We also should be able to sort the list by number of unresolved change requests. Maybe this should be the default sort. So that folks who want to help can see "Oh, this adventure needs a lot of work, let me help."

So,

A: Don't show us resolved change requests, or give us the option to filter them out.

B: Let us sort by number of unresolved change requests so the most grievous offenders get the most attention.

C: Clicking on the adventure title in the list of change requests takes you to the "edit this adventure" page so you can start working.

The smoother we make this process, the more we empower the curators.

**Comment by harkenn** [comment]
@cmfcmf was the plan at some point to replace EasyAdmin if there is time? If so, Curator might be small enough that it can be built without EasyAdmin? That way we can finely control the functionality and UX.

**Comment by cmfcmf** [comment]
> @cmfcmf was the plan at some point to replace EasyAdmin if there is time? If so, Curator might be small enough that it can be built without EasyAdmin? That way we can finely control the functionality and UX.

Yes, removing EasyAdmin would be the end-game. I've been working a bit on a new curation screen, but it's not ready yet.
